### ISSUE TITLE
Will subscribing to YouTube premium fix 403 issues?

### ISSUE THREAD

**Issue by wolfch** [ytdl-org/youtube-dl#32898]
## Checklist

- [x ] I'm asking a question
- [x ] I've looked through the README and FAQ for similar questions
- [ ] I've searched the bugtracker for similar questions including closed ones
^bugtracker blocked by Hamburg Court - I didn't know their jurisdiction was world-wide.

## Question

I'm wondering if subscribing to YouTube premium might help avoiding ad-blocker related issues?

**Comment by bashonly** [comment]
no

**Comment by dirkf** [comment]
Indeed, _yt-dlp_ users say things are worse if you authenticate with cookies.
